numpy/f2py/__init__.py

```diff
@@ -102,7 +101,7 @@ def compile(source,
 
         c = [sys.executable,
              '-c',
-             'import numpy.f2py as f2py2e;f2py2e.main()'] + args
+             'import numpy.f2py as f2pyarg;f2pyarg.main()'] + args
```

Comment:
Why is this symbol renamed here? Perhaps this can also simply import `main` from `f2pyarg` directly?